diff -Nur decopter-0.2.11.orig/common.cc decopter-0.2.11/common.cc
--- decopter-0.2.11.orig/common.cc	2003-03-12 16:18:34.000000000 +0100
+++ decopter-0.2.11/common.cc	2003-05-17 00:21:28.000000000 +0200
@@ -41,7 +41,7 @@
 string GetTexturesDirectory()
 {
         string TexturesDirectory;
-        TexturesDirectory="/usr/local/games/decopter/textures/";
+        TexturesDirectory="/usr/share/decopter/textures/";
         cout <<"TexturesDirectory: "<<TexturesDirectory<<"\n";
         return TexturesDirectory;
 }
diff -Nur decopter-0.2.11.orig/fly.cc decopter-0.2.11/fly.cc
--- decopter-0.2.11.orig/fly.cc	2003-03-12 16:18:34.000000000 +0100
+++ decopter-0.2.11/fly.cc	2003-05-17 00:21:55.000000000 +0200
@@ -24,9 +24,9 @@
 
         gm=new GameManager(InitialScreenWidth,InitialScreenHeight,InitialScreenBPP);
 
-        gm->LoadLandscape("maps/default.land",ViewDistance,40000,MainPatchLevel);
+        gm->LoadLandscape("/usr/share/decopter/maps/default.land",ViewDistance,40000,MainPatchLevel);
 
-        gm->InitCopter("3D/copter.metamesh");
+        gm->InitCopter("/usr/share/decopter/3D/copter.metamesh");
 
         while (!done)
         {
diff -Nur decopter-0.2.11.orig/gamemanager.cc decopter-0.2.11/gamemanager.cc
--- decopter-0.2.11.orig/gamemanager.cc	2003-03-12 16:18:34.000000000 +0100
+++ decopter-0.2.11/gamemanager.cc	2003-05-17 00:23:12.000000000 +0200
@@ -243,7 +243,7 @@
         MetaMeshes["copter"]=MetaMesh(filename);
         MetaMeshes["copter"].GenerateLists();
         copter=new Copter(&(MetaMeshes["copter"]),Vector3D(5000,1200,-15000),landscape);
-        copter->SetTextures("maps/default.copter");
+        copter->SetTextures("/usr/share/decopter/maps/default.copter");
 }
 
 void GameManager::DrawCopter()
diff -Nur decopter-0.2.11.orig/generate_textures.cc decopter-0.2.11/generate_textures.cc
--- decopter-0.2.11.orig/generate_textures.cc	2003-03-12 16:18:34.000000000 +0100
+++ decopter-0.2.11/generate_textures.cc	2003-05-17 00:22:09.000000000 +0200
@@ -3,8 +3,8 @@
 #include "common.h"
 #include <iostream>
 
-#define LandscapeFilename "maps/default.land"
-#define TexturesFilename "maps/default.textures"
+#define LandscapeFilename "/usr/share/decopter/maps/default.land"
+#define TexturesFilename "/usr/share/decopter/maps/default.textures"
 
 #define LightX 0.5
 #define LightZ 0.5
diff -Nur decopter-0.2.11.orig/maps/default.land decopter-0.2.11/maps/default.land
--- decopter-0.2.11.orig/maps/default.land	2003-03-12 16:18:34.000000000 +0100
+++ decopter-0.2.11/maps/default.land	2003-05-17 00:24:48.000000000 +0200
@@ -1,4 +1,4 @@
-map maps/landscape.png
+map /usr/share/decopter/maps/landscape.png
 LandscapeScaleXZ 100.0
 LandscapeScaleY 5000.0
 Light 100000 50000 100000 
diff -Nur decopter-0.2.11.orig/maps/default.trees decopter-0.2.11/maps/default.trees
--- decopter-0.2.11.orig/maps/default.trees	2003-03-12 16:18:34.000000000 +0100
+++ decopter-0.2.11/maps/default.trees	2003-05-17 00:25:11.000000000 +0200
@@ -1,6 +1,6 @@
-textures/tree1.png tree1
-textures/tree2.png tree2
-textures/tree3.png tree3
-textures/tree4.png tree4
-textures/tree5.png tree5
-textures/tree6.png tree6
+/usr/share/decopter/textures/tree1.png tree1
+/usr/share/decopter/textures/tree2.png tree2
+/usr/share/decopter/textures/tree3.png tree3
+/usr/share/decopter/textures/tree4.png tree4
+/usr/share/decopter/textures/tree5.png tree5
+/usr/share/decopter/textures/tree6.png tree6
